fairlearn/metrics_plots/plot_disparities_in_metric.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, y_true, y_pred, sensitive_features, show_plot=True):
```

Comment:
```suggestion
def plot_disparities_in_metric(metric, *, y_true, y_pred, sensitive_features, show_plot=True):
```

Reply:
Added this in both functions. (didn't take suggestion to avoid DCO issues)